sdk/tests/conformance/resources/tex-image-and-sub-image-2d-with-webgl-canvas.js

```diff
@@ -26,6 +26,9 @@ function generateTest(pixelFormat, pixelType, prologue) {
     var gl = null;
     var successfullyParsed = false;
 
+    var TEXTURE_2D = 0;
+    var TEXTURE_CUBE_MAP = 1;
+
```

Comment:
Is it necessary to create these two names?  It seems you can just use gl.TEXTURE_2D and gl.TEXTURE_CUBE_MAP as gl is initialized right in the beginning.